### PR TITLE
Add journal seek latency to grafana dashboard

### PR DESCRIPTION
**Pull request by deepthidevaki**
## Description

In PR #12869 we added metrics for journal seek. This PR add a panel for journal seek metrics in dashboard zeebe.json

Since the raft section is too big, all metrics related to journal is moved to a new section named "Journal". Metrics for seek is added under this section.

### CODE REVIEW

**Comment by deepthidevaki** [comment]
You can view the dashboard here https://grafana.dev.zeebe.io/d/I4lo7_EZk_dd/temp-zeebe-journal-seek?orgId=1&refresh=10s&var-DS_PROMETHEUS=default&var-cluster=All&var-namespace=All&var-pod=All&var-partition=All&from=now-15m&to=now 


New row "Journal" is added after "Raft"
image

image

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -12016,13 +12232,13 @@
     {
       "collapsed": true,
       "datasource": {
-        "uid": "${DS_PROMETHEUS}"
+        "uid": "prometheus"
```
Comment (npepinpe) [inline monitor/grafana/zeebe.json:12235]: :x: Should be kept as `${DS_PROMETHEUS}`

**Review by npepinpe — APPROVED** [review]
:x: There's a bunch of changes which fix the UID of the datasource to `prometheus` instead of `${DS_PROMETHEUS}`, which is not portable. Just do a quick search-replace for it. I was going to highlight them but there's too many, and it felt silly to write a comment for each :smile: 

No need for another review. We just need to make sure we don't hardcode UIDs

**Comment by deepthidevaki** [comment]
bors merge

**Pull request merged**